arc/aws/391835788720/us-east-1/lf-arc-prod-eks.tf

```diff
@@ -0,0 +1,76 @@
+module "pytorch_arc_prod_eks" {
```

Comment:
We should call this a `dev` system rather than prod since we won't be deploying a prod level system for awhile.